### ISSUE TITLE
how do i use it

### ISSUE THREAD

**Issue by NathanMYname** [Luckytrang2010/BrutePin#16]
i git cloned it and when i do ./main.py this comes up
![image](https://user-images.githubusercontent.com/62193238/121790287-7a7c3400-cba3-11eb-952f-5d08d541883b.png)

**Comment by DBooker18** [comment]
instead of "./main.py" try "python3 main.py" make sure u install requirements first!

**Comment by NathanMYname** [comment]
> instead of "./main.py" try "python3 main.py" make sure u install requirements first!

ok thanks

**Comment by Xpoboss** [comment]
> instead of "./main.py" try "python3 main.py" make sure u install requirements first!

![1231231](https://user-images.githubusercontent.com/86512842/124202107-52476d00-da9f-11eb-84cf-2e88f1af78e0.png)

**Comment by Xpoboss** [comment]
> instead of "./main.py" try "python3 main.py" make sure u install requirements first!

how do i install the requirements?

**Comment by Xpoboss** [comment]
![1231231](https://user-images.githubusercontent.com/86512842/124204368-8d986a80-daa4-11eb-9d16-80a99937ae08.png)

> instead of "./main.py" try "python3 main.py" make sure u install requirements first!

im still getting this message any reason why?

**Comment by DBooker18** [comment]
yes, you have to install "requests" module to do that type "pip3 install requests" then run "python3 main.py"
